llmfoundry/models/hf/hf_base.py

```diff
@@ -69,7 +69,7 @@ def __init__(
         config_overrides: Optional[dict[str, Any]] = None,
         use_logits: bool = True,
         shift_labels: bool = False,
-        peft_config: Optional['PeftConfig'] = None,
```

Comment:
why no typing?

Reply:
this typing was just wrong. this function accepts a dict and then converts it to a peft config